scripts/fetch/fetchPrecip.R

```diff
@@ -37,9 +36,9 @@ fetch.precip <- function(viz){
   endDate <- as.POSIXct(paste(viz[["end.date"]],"22:00:00"), tz="America/New_York")
   attr(startDate, 'tzone') <- "UTC"
   
-  states <- viz[['states']]
+  counties <- readDepends(viz)[['counties']] #TODO: need to get counties, send into getPrecip
```

Comment:
still foggy on the depends naming. this works when you keep the depends in an array:
```r
counties <- readDepends(viz)[['counties']]
```

Reply:
so why do we need to name?

Reply:
just talked about this in person, but for future reference -- only time we need a name is when we have a generic function that gets used with multiple depends. E.g. a processor step uses `dailyData` as input but we use the processor step two different times with different data; we would need something generic so the processor function could use whichever data actually came in.